### ISSUE TITLE
[Bug] AutomationProperties.LabeledByProperty has no effect when using VoiceOver on iOS (Android and UWP are fine)

### ISSUE THREAD

**Issue by johnshardman** [xamarin/Xamarin.Forms#12516]
Using XF 4.8, the AutomationProperties.LabeledByProperty property seems to be having no effect on iOS.
Whereas on UWP and Android, it results in the Text property or the accessibility information from the referenced View being read out by TalkBack or Narrator, on iOS VoiceOver does not read information from the referenced View.

I've checked this with Slider, Picker and Switch. 

e.g. In the case of Slider, the following code should result in VoiceOver reading the label information when interacting with the slider. However, it doesn't.

slider.SetValue(AutomationProperties.LabeledByProperty, label);

### Basic Information

- Version with issue: XF 4.8
- iOS:  14.0.1

**Comment by rachelkang** [comment]
@johnshardman thanks for submitting this issue! It looks like we currently have no implementation of `AutomationProperties.LabeledByProperty` for iOS :(  We'll be updating our docs to reflect this gap, and will be sure to explore this issue further

**Comment by Burgler-dev** [comment]
We also need this, this is essential for good accessibility support.

**Comment by rachelkang** [comment]
Upon further investigation, we have learned much more about the `LabeledBy` API as tracked in https://github.com/dotnet/maui/issues/845. On Android and Windows, we've found there is little to no difference between the behavior of `LabeledBy` and `AutomationProperties.Name` (or `SemanticProperties.Description`) and this API had no iOS implementation to begin with because there is no platform equivalent on iOS. Therefore, there are no current plans to implement this
